### ISSUE TITLE
BUG: update_configuration for lambda is trying to set a property

### ISSUE THREAD

**Issue by SophieMJ** [spulec/moto#3464]
## Reporting Bugs

When using moto for testing `update_function_configuration`, this block of code is being called: https://github.com/spulec/moto/blob/47dbad291e717059a93f5cf795a9be08cfc872cc/moto/awslambda/models.py#L293-L310

However, this is giving an error provided below:
```
elif key == “VpcConfig”:
       self.vpc_config = value
              AttributeError: can’t set attribute

../../.virtualenvs/lambda-zvljBF5N/lib/python3.8/site-packages/moto/awslambda/models.py:323: AttributeError
```

This is because on L308 it's trying to set a value to `vpc_config` which is a property (not an attribute) that doesn't have a setter. I think changing it to `self._vpc_config = value` should fix the problem, but any suggestions/comments are welcome.

**Comment by bblommers** [comment]
Hey @SophieMJ, thanks for raising this! Looks like that should be as simple as changing it to `_vpc_config`, yes. 
Marking it as a bug.

**Comment by bblommers** [comment]
This is now fixed as of moto >= 1.3.16.dev130
